markdown/events/2022/hackathon-march-2022.md

```diff
@@ -99,6 +101,15 @@ This group will focus on restructuring and organising the documentation on the w
 - Restructure website navigation and improve the user experience.
 - Act in a support role for technical help to others in the hackathon.
 
+## Talks
+
+We will also be offering three different talks throughout the event at 14:30 each day.
+These focus on useful 'soft skills' for developers that can enhance their documentation.
+
+🖉 "Draw stuff good: how to use Inkscape" by [Zandra Fagernäs](https://twitter.com/ZandraSelina)<br>
```

Comment:
this emoji doesn't seem to get rendered in github nor gitpod.

Reply:
@mashehu updated, can youcheck again?